5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM rhel7
```

Comment:
then we need the openshift/base-rhel7 in OSBS....  
@sdodson how are you dealing with this today?